deployer/develop/task/prepare_release.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+namespace Deployer;
+
+use Deployer\Exception\Exception;
+use Symfony\Component\Console\Helper\Table;
+
+task('develop:release:prepare', function () {
+    /* get new version */
+    if (is_null(input()->getOption('newVersion'))) {
+        throw new Exception('Missing option "version" for new release', 1711458053);
+    }
+    $version = input()->getOption('newVersion');
+    set('newVersion', $version);
+
+    checkPreconditions();
+
+    info("The following steps will be executed:\n - tabula_rasa \n - start_new_release \n - " . implode("\n - ", get('develop_steps')));
```

Comment:
Has to be `develop_release_steps` else throws errors.

Reply:
I removed this steps.